<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HTTP</class>
 <widget class="QWidget" name="HTTP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HTTP</string>
  </property>
  <widget class="QPushButton" name="pushButton_image">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>360</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pushButton_image
{
    border: none;       /*无边框*/
    background: transparent;/*背景透明*/
    border-radius: 15px;/*圆角半径（根据按钮大小调整）*/
}
QPushButton#pushButton_image:hover 
{
    background-color: lightgreen;/*悬停时为绿色*/
}
QPushButton#pushButton_image:pressed 
{
     background-color: darkgreen;/*按下时为深绿色*/
}</string>
   </property>
   <property name="text">
    <string>生成</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_image">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>431</width>
     <height>321</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>360</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pushButton
{
    border: none;       /*无边框*/
    background: transparent;/*背景透明*/
    border-radius: 15px;/*圆角半径（根据按钮大小调整）*/
}
QPushButton#pushButton:hover 
{
    background-color: lightgreen;/*悬停时为绿色*/
}
QPushButton#pushButton:pressed 
{
     background-color: darkgreen;/*按下时为深绿色*/
}</string>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_send">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>361</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_reply">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>361</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit#textEdit_reply
{
    border: none;       /*无边框*/
    background: transparent;/*背景透明*/
}</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
